### DISCUSSION TITLE
Documentation site

### DISCUSSION THREAD

**Discussion by dishwasher-detergent** [sanny-io/react-appwrite#26] [Ideas]
Should we create a documentation site? I've got an idea up and running, lmk what you think

https://react-appwrite-docs.vercel.app

**Comment by sanny-io** [comment]
We should! I bought a domain yesterday.

Bro this thing looks amazing already. Did you make this corndocs project???? Cause I'm telling you bro this is seriously awesome. How can we get it deployed? Like where is the repo?

**Reply by dishwasher-detergent** [reply]
Yea, I made the corndocs project a bit ago. Looking back it's not the best code but it gets the job done lol

**Reply by dishwasher-detergent** [reply]
I currently have it in https://github.com/dishwasher-detergent/react-appwrite-docs, but it can be moved to the organization. You can deploy it to vercel pretty quickly.

**Reply by sanny-io** [reply]
I've moved it into the organization. I'm currently working on getting Vercel sponsorship so we can host it there without paying.

**Reply by sanny-io** [reply]
@dishwasher-detergent so the site is live at https://react-appwrite.org, but when I run `next dev` locally it infinitely loads the page, and when I run `next build` it gets stuck like this.

![](https://i.imgur.com/y2mKJza.png)

Do you have any ideas what might be causing this? It's difficult to make adjustments without being able to see them.

**Reply by dishwasher-detergent** [reply]
Hmm, you should only have to do npm install and npm run dev.

**Reply by dishwasher-detergent** [reply]
@sanny-io Not sure why you're getting any issues. I've cloned the new repo you made, ran npm run dev and npm run build with no issues.

**Reply by dishwasher-detergent** [reply]
Also I did notice a hydration issue with, what I believe to be, the codeblock component. I'll look into it sometime this week.

**Reply by dishwasher-detergent** [reply]
@sanny-io I am running Windows 10. The only global npm packages I have currently are create-react-app, and angular.

**Reply by dishwasher-detergent** [reply]
Also, glob comes with the glob-promise package

**Reply by sanny-io** [reply]
Yes you are right, sorry. I don't know why I'm having problems with this glob stuff. In any case, I have a fix for the infinite loading and will push it later tonight.
